Formula/texlive.rb

```diff
@@ -0,0 +1,161 @@
+class Texlive < Formula
+  desc "Free software distribution for the TeX typesetting system"
+  homepage "https://www.tug.org/texlive/"
+  url "https://github.com/TeX-Live/texlive-source/archive/refs/tags/svn58837.tar.gz"
+  sha256 "0afa6919e44675b7afe0fa45344747afef07b6ee98eeb14ff6a2ef78f458fc12"
+  license :public_domain
+  head "https://github.com/TeX-Live/texlive-source.git", branch: "trunk"
+
+  depends_on "cairo"
+  depends_on "clisp"
+  depends_on "fontconfig"
+  depends_on "freetype"
+  depends_on "gd"
+  depends_on "ghostscript"
+  depends_on "gmp"
+  depends_on "graphite2"
+  depends_on "harfbuzz"
+  depends_on "libpng"
+  depends_on "lua"
+  depends_on "luajit-openresty"
+  depends_on "mpfr"
+  depends_on "perl"
+  depends_on "pixman"
+  depends_on "potrace"
+
+  uses_from_macos "icu4c"
+  uses_from_macos "ncurses"
+  uses_from_macos "zlib"
+
+  on_linux do
+    depends_on "pkg-config" => :build
+    depends_on "gcc"
+    depends_on "libice"
+    depends_on "libsm"
+    depends_on "libx11"
+    depends_on "libxaw"
+    depends_on "libxext"
+    depends_on "libxmu"
+    depends_on "libxpm"
+    depends_on "libxt"
+  end
+
+  fails_with gcc: "5"
+
+  resource("texlive-extra") do
+    url "https://ftp.math.utah.edu/pub/tex/historic/systems/texlive/2021/texlive-20210325-extra.tar.xz"
+    sha256 "46a3f385d0b30893eec6b39352135d2929ee19a0a81df2441bfcaa9f6c78339c"
+  end
+
+  resource "install-tl" do
+    url "https://ftp.math.utah.edu/pub/tex/historic/systems/texlive/2021/install-tl-unx.tar.gz"
+    sha256 "74eac0855e1e40c8db4f28b24ef354bd7263c1f76031bdc02b52156b572b7a1d"
+  end
+
+  resource("texlive-texmf") do
+    url "https://ftp.math.utah.edu/pub/tex/historic/systems/texlive/2021/texlive-20210325-texmf.tar.xz"
+    sha256 "ff12d436c23e99fb30aad55924266104356847eb0238c193e839c150d9670f1c"
+  end
+
+  def install
+    # Install resources
+    resource("texlive-extra").stage do
+      (libexec/"share").install "tlpkg"
+    end
+
+    resource("install-tl").stage do
+      cd "tlpkg" do
+        (libexec/"share/tlpkg").install "installer"
+      end
+    end
+
+    resource("texlive-texmf").stage do
+      (libexec/"share").install "texmf-dist"
+    end
+
+    # Clean unused files
+    rm_rf libexec/"share/texmf-dist/doc"
+    rm_rf libexec/"share/tlpkg/installer/wget"
+    rm_rf libexec/"share/tlpkg/installer/xz"
+
+    args = [
+      "--prefix=#{libexec}",
+      "--disable-debug",
+      "--disable-dependency-tracking",
+      "--disable-dvisvgm", # needs its own formula
+      "--disable-missing",
+      "--disable-native-texlive-build", # needed when doing a distro build
+      "--disable-ps2eps", # provided by ps2eps formula
+      "--disable-psutils", # provided by psutils formula
+      "--disable-static",
+      "--disable-t1utils", # provided by t1utils formula
+      "--enable-build-in-source-tree",
+      "--enable-shared",
+      "--enable-compiler-warnings=yes",
+      "--with-banner-add=#{tap.user}",
```

Comment:
maybe?
```suggestion
      "--with-banner-add=/#{tap.user}",
```
otherwise it seems off

```
$ tex --version
TeX 3.141592653 (TeX Live 2021Homebrew)
```

Reply:
That sounds like an upstream issue to me.

Reply:
seems to be [intentional](http://tug.org/texlive/doc/tlbuild.html#Distro-builds)

> --with-banner-add=/SomeDistro

Reply:
Yea, definitely seems intentional. It gives you more control over the version scheme of your distribution if you want it.